2022-06-24 12:23:07 Note: NumExpr detected 12 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.
2022-06-24 12:23:07 NumExpr defaulting to 8 threads.
2022-06-24 12:23:09 Falling back to TensorFlow client; we recommended you install the Cloud TPU client directly with pip install cloud-tpu-client.
2022-06-24 12:23:10 Creating converter from 7 to 5
2022-06-24 12:23:10 Creating converter from 5 to 7
2022-06-24 12:23:10 Creating converter from 7 to 5
2022-06-24 12:23:10 Creating converter from 5 to 7
2022-06-24 12:23:55 addr2: cardinality : 74 and embedding dim: 37
2022-06-24 12:23:58 id_06: cardinality : 103 and embedding dim: 51
2022-06-24 12:23:58 id_33: cardinality : 262 and embedding dim: 100
2022-06-24 12:23:58 M4: cardinality : 5 and embedding dim: 2
2022-06-24 12:23:58 DeviceInfo: cardinality : 1788 and embedding dim: 100
2022-06-24 12:23:58 id_25: cardinality : 343 and embedding dim: 100
2022-06-24 12:23:58 id_22: cardinality : 27 and embedding dim: 13
2022-06-24 12:23:58 card4: cardinality : 6 and embedding dim: 3
2022-06-24 12:23:58 id_31: cardinality : 132 and embedding dim: 66
2022-06-24 12:23:58 id_26: cardinality : 97 and embedding dim: 48
2022-06-24 12:23:58 id_32: cardinality : 6 and embedding dim: 3
2022-06-24 12:23:58 id_01: cardinality : 78 and embedding dim: 39
2022-06-24 12:23:58 card1: cardinality : 8500 and embedding dim: 100
2022-06-24 12:23:58 id_04: cardinality : 17 and embedding dim: 8
2022-06-24 12:23:58 id_08: cardinality : 96 and embedding dim: 48
2022-06-24 12:23:58 R_emaildomain: cardinality : 62 and embedding dim: 31
2022-06-24 12:23:58 id_05: cardinality : 95 and embedding dim: 47
2022-06-24 12:23:58 id_24: cardinality : 14 and embedding dim: 7
2022-06-24 12:23:58 id_15: cardinality : 5 and embedding dim: 2
2022-06-24 12:23:58 ProductCD: cardinality : 5 and embedding dim: 2
2022-06-24 12:23:58 id_23: cardinality : 5 and embedding dim: 2
2022-06-24 12:23:58 id_13: cardinality : 56 and embedding dim: 28
2022-06-24 12:23:58 id_17: cardinality : 106 and embedding dim: 53
2022-06-24 12:23:58 card2: cardinality : 484 and embedding dim: 100
2022-06-24 12:23:58 card5: cardinality : 108 and embedding dim: 54
2022-06-24 12:23:58 id_10: cardinality : 64 and embedding dim: 32
2022-06-24 12:23:58 id_03: cardinality : 26 and embedding dim: 13
2022-06-24 12:23:58 id_07: cardinality : 86 and embedding dim: 43
2022-06-24 12:23:58 id_14: cardinality : 27 and embedding dim: 13
2022-06-24 12:23:58 id_18: cardinality : 20 and embedding dim: 10
2022-06-24 12:23:58 id_19: cardinality : 524 and embedding dim: 100
2022-06-24 12:23:58 card6: cardinality : 5 and embedding dim: 2
2022-06-24 12:23:58 id_21: cardinality : 492 and embedding dim: 100
2022-06-24 12:23:58 id_30: cardinality : 77 and embedding dim: 38
2022-06-24 12:23:58 id_34: cardinality : 6 and embedding dim: 3
2022-06-24 12:23:58 card3: cardinality : 105 and embedding dim: 52
2022-06-24 12:23:59 id_20: cardinality : 396 and embedding dim: 100
2022-06-24 12:23:59 id_09: cardinality : 48 and embedding dim: 24
2022-06-24 12:23:59 id_11: cardinality : 367 and embedding dim: 100
2022-06-24 12:23:59 addr1: cardinality : 257 and embedding dim: 100
2022-06-24 12:23:59 P_emaildomain: cardinality : 61 and embedding dim: 30
2022-06-24 12:23:59 None
2022-06-24 12:24:26 Only one class present in y_true. ROC AUC score is not defined in that case.
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Temp\tmpt5n54rjz.py", line 19, in fallback_auc
    retval__1 = ag__.converted_call(ag__.ld(roc_auc_score), (ag__.ld(y_true_), ag__.ld(y_pred_)), None, fscope_1)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 340, in converted_call
    return _call_unconverted(f, args, kwargs, options, False)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 464, in _call_unconverted
    return f(*args)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 569, in roc_auc_score
    return _average_binary_score(
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_base.py", line 75, in _average_binary_score
    return binary_metric(y_true, y_score, sample_weight=sample_weight)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 338, in _binary_roc_auc_score
    raise ValueError(
ValueError: Only one class present in y_true. ROC AUC score is not defined in that case.
2022-06-24 12:24:28 Only one class present in y_true. ROC AUC score is not defined in that case.
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Temp\tmpt5n54rjz.py", line 19, in fallback_auc
    retval__1 = ag__.converted_call(ag__.ld(roc_auc_score), (ag__.ld(y_true_), ag__.ld(y_pred_)), None, fscope_1)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 340, in converted_call
    return _call_unconverted(f, args, kwargs, options, False)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 464, in _call_unconverted
    return f(*args)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 569, in roc_auc_score
    return _average_binary_score(
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_base.py", line 75, in _average_binary_score
    return binary_metric(y_true, y_score, sample_weight=sample_weight)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 338, in _binary_roc_auc_score
    raise ValueError(
ValueError: Only one class present in y_true. ROC AUC score is not defined in that case.
2022-06-24 12:24:51 addr2: cardinality : 74 and embedding dim: 37
2022-06-24 12:24:51 id_06: cardinality : 103 and embedding dim: 51
2022-06-24 12:24:51 id_33: cardinality : 262 and embedding dim: 100
2022-06-24 12:24:51 M4: cardinality : 5 and embedding dim: 2
2022-06-24 12:24:51 DeviceInfo: cardinality : 1788 and embedding dim: 100
2022-06-24 12:24:51 id_25: cardinality : 343 and embedding dim: 100
2022-06-24 12:24:51 id_22: cardinality : 27 and embedding dim: 13
2022-06-24 12:24:51 card4: cardinality : 6 and embedding dim: 3
2022-06-24 12:24:51 id_31: cardinality : 132 and embedding dim: 66
2022-06-24 12:24:51 id_26: cardinality : 97 and embedding dim: 48
2022-06-24 12:24:51 id_32: cardinality : 6 and embedding dim: 3
2022-06-24 12:24:51 id_01: cardinality : 78 and embedding dim: 39
2022-06-24 12:24:51 card1: cardinality : 8500 and embedding dim: 100
2022-06-24 12:24:51 id_04: cardinality : 17 and embedding dim: 8
2022-06-24 12:24:51 id_08: cardinality : 96 and embedding dim: 48
2022-06-24 12:24:51 R_emaildomain: cardinality : 62 and embedding dim: 31
2022-06-24 12:24:51 id_05: cardinality : 95 and embedding dim: 47
2022-06-24 12:24:51 id_24: cardinality : 14 and embedding dim: 7
2022-06-24 12:24:51 id_15: cardinality : 5 and embedding dim: 2
2022-06-24 12:24:51 ProductCD: cardinality : 5 and embedding dim: 2
2022-06-24 12:24:51 id_23: cardinality : 5 and embedding dim: 2
2022-06-24 12:24:51 id_13: cardinality : 56 and embedding dim: 28
2022-06-24 12:24:51 id_17: cardinality : 106 and embedding dim: 53
2022-06-24 12:24:51 card2: cardinality : 484 and embedding dim: 100
2022-06-24 12:24:51 card5: cardinality : 108 and embedding dim: 54
2022-06-24 12:24:51 id_10: cardinality : 64 and embedding dim: 32
2022-06-24 12:24:51 id_03: cardinality : 26 and embedding dim: 13
2022-06-24 12:24:51 id_07: cardinality : 86 and embedding dim: 43
2022-06-24 12:24:51 id_14: cardinality : 27 and embedding dim: 13
2022-06-24 12:24:51 id_18: cardinality : 20 and embedding dim: 10
2022-06-24 12:24:51 id_19: cardinality : 524 and embedding dim: 100
2022-06-24 12:24:51 card6: cardinality : 5 and embedding dim: 2
2022-06-24 12:24:51 id_21: cardinality : 492 and embedding dim: 100
2022-06-24 12:24:51 id_30: cardinality : 77 and embedding dim: 38
2022-06-24 12:24:51 id_34: cardinality : 6 and embedding dim: 3
2022-06-24 12:24:51 card3: cardinality : 105 and embedding dim: 52
2022-06-24 12:24:51 id_20: cardinality : 396 and embedding dim: 100
2022-06-24 12:24:51 id_09: cardinality : 48 and embedding dim: 24
2022-06-24 12:24:51 id_11: cardinality : 367 and embedding dim: 100
2022-06-24 12:24:51 addr1: cardinality : 257 and embedding dim: 100
2022-06-24 12:24:51 P_emaildomain: cardinality : 61 and embedding dim: 30
2022-06-24 12:24:52 None
2022-06-24 12:25:16 Only one class present in y_true. ROC AUC score is not defined in that case.
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Temp\tmpt5n54rjz.py", line 19, in fallback_auc
    retval__1 = ag__.converted_call(ag__.ld(roc_auc_score), (ag__.ld(y_true_), ag__.ld(y_pred_)), None, fscope_1)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 340, in converted_call
    return _call_unconverted(f, args, kwargs, options, False)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 464, in _call_unconverted
    return f(*args)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 569, in roc_auc_score
    return _average_binary_score(
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_base.py", line 75, in _average_binary_score
    return binary_metric(y_true, y_score, sample_weight=sample_weight)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 338, in _binary_roc_auc_score
    raise ValueError(
ValueError: Only one class present in y_true. ROC AUC score is not defined in that case.
2022-06-24 12:25:18 Only one class present in y_true. ROC AUC score is not defined in that case.
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Temp\tmpt5n54rjz.py", line 19, in fallback_auc
    retval__1 = ag__.converted_call(ag__.ld(roc_auc_score), (ag__.ld(y_true_), ag__.ld(y_pred_)), None, fscope_1)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 340, in converted_call
    return _call_unconverted(f, args, kwargs, options, False)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 464, in _call_unconverted
    return f(*args)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 569, in roc_auc_score
    return _average_binary_score(
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_base.py", line 75, in _average_binary_score
    return binary_metric(y_true, y_score, sample_weight=sample_weight)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 338, in _binary_roc_auc_score
    raise ValueError(
ValueError: Only one class present in y_true. ROC AUC score is not defined in that case.
2022-06-24 12:25:37 Only one class present in y_true. ROC AUC score is not defined in that case.
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Temp\tmpt5n54rjz.py", line 19, in fallback_auc
    retval__1 = ag__.converted_call(ag__.ld(roc_auc_score), (ag__.ld(y_true_), ag__.ld(y_pred_)), None, fscope_1)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 340, in converted_call
    return _call_unconverted(f, args, kwargs, options, False)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 464, in _call_unconverted
    return f(*args)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 569, in roc_auc_score
    return _average_binary_score(
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_base.py", line 75, in _average_binary_score
    return binary_metric(y_true, y_score, sample_weight=sample_weight)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 338, in _binary_roc_auc_score
    raise ValueError(
ValueError: Only one class present in y_true. ROC AUC score is not defined in that case.
2022-06-24 12:25:39 Only one class present in y_true. ROC AUC score is not defined in that case.
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Temp\tmpt5n54rjz.py", line 19, in fallback_auc
    retval__1 = ag__.converted_call(ag__.ld(roc_auc_score), (ag__.ld(y_true_), ag__.ld(y_pred_)), None, fscope_1)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 340, in converted_call
    return _call_unconverted(f, args, kwargs, options, False)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 464, in _call_unconverted
    return f(*args)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 569, in roc_auc_score
    return _average_binary_score(
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_base.py", line 75, in _average_binary_score
    return binary_metric(y_true, y_score, sample_weight=sample_weight)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 338, in _binary_roc_auc_score
    raise ValueError(
ValueError: Only one class present in y_true. ROC AUC score is not defined in that case.
2022-06-24 12:25:57 Only one class present in y_true. ROC AUC score is not defined in that case.
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Temp\tmpt5n54rjz.py", line 19, in fallback_auc
    retval__1 = ag__.converted_call(ag__.ld(roc_auc_score), (ag__.ld(y_true_), ag__.ld(y_pred_)), None, fscope_1)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 340, in converted_call
    return _call_unconverted(f, args, kwargs, options, False)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 464, in _call_unconverted
    return f(*args)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 569, in roc_auc_score
    return _average_binary_score(
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_base.py", line 75, in _average_binary_score
    return binary_metric(y_true, y_score, sample_weight=sample_weight)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 338, in _binary_roc_auc_score
    raise ValueError(
ValueError: Only one class present in y_true. ROC AUC score is not defined in that case.
2022-06-24 12:26:00 Only one class present in y_true. ROC AUC score is not defined in that case.
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Temp\tmpt5n54rjz.py", line 19, in fallback_auc
    retval__1 = ag__.converted_call(ag__.ld(roc_auc_score), (ag__.ld(y_true_), ag__.ld(y_pred_)), None, fscope_1)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 340, in converted_call
    return _call_unconverted(f, args, kwargs, options, False)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 464, in _call_unconverted
    return f(*args)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 569, in roc_auc_score
    return _average_binary_score(
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_base.py", line 75, in _average_binary_score
    return binary_metric(y_true, y_score, sample_weight=sample_weight)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 338, in _binary_roc_auc_score
    raise ValueError(
ValueError: Only one class present in y_true. ROC AUC score is not defined in that case.
2022-06-24 12:26:18 Only one class present in y_true. ROC AUC score is not defined in that case.
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Temp\tmpt5n54rjz.py", line 19, in fallback_auc
    retval__1 = ag__.converted_call(ag__.ld(roc_auc_score), (ag__.ld(y_true_), ag__.ld(y_pred_)), None, fscope_1)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 340, in converted_call
    return _call_unconverted(f, args, kwargs, options, False)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 464, in _call_unconverted
    return f(*args)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 569, in roc_auc_score
    return _average_binary_score(
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_base.py", line 75, in _average_binary_score
    return binary_metric(y_true, y_score, sample_weight=sample_weight)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 338, in _binary_roc_auc_score
    raise ValueError(
ValueError: Only one class present in y_true. ROC AUC score is not defined in that case.
2022-06-24 12:26:20 Only one class present in y_true. ROC AUC score is not defined in that case.
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Temp\tmpt5n54rjz.py", line 19, in fallback_auc
    retval__1 = ag__.converted_call(ag__.ld(roc_auc_score), (ag__.ld(y_true_), ag__.ld(y_pred_)), None, fscope_1)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 340, in converted_call
    return _call_unconverted(f, args, kwargs, options, False)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 464, in _call_unconverted
    return f(*args)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 569, in roc_auc_score
    return _average_binary_score(
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_base.py", line 75, in _average_binary_score
    return binary_metric(y_true, y_score, sample_weight=sample_weight)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 338, in _binary_roc_auc_score
    raise ValueError(
ValueError: Only one class present in y_true. ROC AUC score is not defined in that case.
2022-06-24 12:26:39 Only one class present in y_true. ROC AUC score is not defined in that case.
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Temp\tmpt5n54rjz.py", line 19, in fallback_auc
    retval__1 = ag__.converted_call(ag__.ld(roc_auc_score), (ag__.ld(y_true_), ag__.ld(y_pred_)), None, fscope_1)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 340, in converted_call
    return _call_unconverted(f, args, kwargs, options, False)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 464, in _call_unconverted
    return f(*args)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 569, in roc_auc_score
    return _average_binary_score(
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_base.py", line 75, in _average_binary_score
    return binary_metric(y_true, y_score, sample_weight=sample_weight)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 338, in _binary_roc_auc_score
    raise ValueError(
ValueError: Only one class present in y_true. ROC AUC score is not defined in that case.
2022-06-24 12:26:41 Only one class present in y_true. ROC AUC score is not defined in that case.
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Temp\tmpt5n54rjz.py", line 19, in fallback_auc
    retval__1 = ag__.converted_call(ag__.ld(roc_auc_score), (ag__.ld(y_true_), ag__.ld(y_pred_)), None, fscope_1)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 340, in converted_call
    return _call_unconverted(f, args, kwargs, options, False)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 464, in _call_unconverted
    return f(*args)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 569, in roc_auc_score
    return _average_binary_score(
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_base.py", line 75, in _average_binary_score
    return binary_metric(y_true, y_score, sample_weight=sample_weight)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 338, in _binary_roc_auc_score
    raise ValueError(
ValueError: Only one class present in y_true. ROC AUC score is not defined in that case.
2022-06-24 12:27:00 Only one class present in y_true. ROC AUC score is not defined in that case.
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Temp\tmpt5n54rjz.py", line 19, in fallback_auc
    retval__1 = ag__.converted_call(ag__.ld(roc_auc_score), (ag__.ld(y_true_), ag__.ld(y_pred_)), None, fscope_1)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 340, in converted_call
    return _call_unconverted(f, args, kwargs, options, False)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 464, in _call_unconverted
    return f(*args)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 569, in roc_auc_score
    return _average_binary_score(
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_base.py", line 75, in _average_binary_score
    return binary_metric(y_true, y_score, sample_weight=sample_weight)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 338, in _binary_roc_auc_score
    raise ValueError(
ValueError: Only one class present in y_true. ROC AUC score is not defined in that case.
2022-06-24 12:27:02 Only one class present in y_true. ROC AUC score is not defined in that case.
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Temp\tmpt5n54rjz.py", line 19, in fallback_auc
    retval__1 = ag__.converted_call(ag__.ld(roc_auc_score), (ag__.ld(y_true_), ag__.ld(y_pred_)), None, fscope_1)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 340, in converted_call
    return _call_unconverted(f, args, kwargs, options, False)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 464, in _call_unconverted
    return f(*args)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 569, in roc_auc_score
    return _average_binary_score(
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_base.py", line 75, in _average_binary_score
    return binary_metric(y_true, y_score, sample_weight=sample_weight)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 338, in _binary_roc_auc_score
    raise ValueError(
ValueError: Only one class present in y_true. ROC AUC score is not defined in that case.
2022-06-24 12:27:21 Only one class present in y_true. ROC AUC score is not defined in that case.
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Temp\tmpt5n54rjz.py", line 19, in fallback_auc
    retval__1 = ag__.converted_call(ag__.ld(roc_auc_score), (ag__.ld(y_true_), ag__.ld(y_pred_)), None, fscope_1)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 340, in converted_call
    return _call_unconverted(f, args, kwargs, options, False)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 464, in _call_unconverted
    return f(*args)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 569, in roc_auc_score
    return _average_binary_score(
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_base.py", line 75, in _average_binary_score
    return binary_metric(y_true, y_score, sample_weight=sample_weight)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 338, in _binary_roc_auc_score
    raise ValueError(
ValueError: Only one class present in y_true. ROC AUC score is not defined in that case.
2022-06-24 12:27:23 Only one class present in y_true. ROC AUC score is not defined in that case.
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Temp\tmpt5n54rjz.py", line 19, in fallback_auc
    retval__1 = ag__.converted_call(ag__.ld(roc_auc_score), (ag__.ld(y_true_), ag__.ld(y_pred_)), None, fscope_1)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 340, in converted_call
    return _call_unconverted(f, args, kwargs, options, False)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\tensorflow\python\autograph\impl\api.py", line 464, in _call_unconverted
    return f(*args)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 569, in roc_auc_score
    return _average_binary_score(
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_base.py", line 75, in _average_binary_score
    return binary_metric(y_true, y_score, sample_weight=sample_weight)
  File "C:\Users\user\anaconda3\envs\tf_gpu\lib\site-packages\sklearn\metrics\_ranking.py", line 338, in _binary_roc_auc_score
    raise ValueError(
ValueError: Only one class present in y_true. ROC AUC score is not defined in that case.
2022-06-24 12:27:24 addr2 dim: 37
2022-06-24 12:27:24 id_06 dim: 51
2022-06-24 12:27:24 id_33 dim: 100
2022-06-24 12:27:24 M4 dim: 2
2022-06-24 12:27:24 DeviceInfo dim: 100
2022-06-24 12:27:24 id_25 dim: 100
2022-06-24 12:27:24 id_22 dim: 13
2022-06-24 12:27:24 card4 dim: 3
2022-06-24 12:27:24 id_31 dim: 66
2022-06-24 12:27:24 id_26 dim: 48
2022-06-24 12:27:24 id_32 dim: 3
2022-06-24 12:27:24 id_01 dim: 39
2022-06-24 12:27:24 card1 dim: 100
2022-06-24 12:27:24 id_04 dim: 8
2022-06-24 12:27:24 id_08 dim: 48
2022-06-24 12:27:24 R_emaildomain dim: 31
2022-06-24 12:27:24 id_05 dim: 47
2022-06-24 12:27:24 id_24 dim: 7
2022-06-24 12:27:24 id_15 dim: 2
2022-06-24 12:27:24 ProductCD dim: 2
2022-06-24 12:27:24 id_23 dim: 2
2022-06-24 12:27:24 id_13 dim: 28
2022-06-24 12:27:24 id_17 dim: 53
2022-06-24 12:27:24 card2 dim: 100
2022-06-24 12:27:24 card5 dim: 54
2022-06-24 12:27:24 id_10 dim: 32
2022-06-24 12:27:24 id_03 dim: 13
2022-06-24 12:27:24 id_07 dim: 43
2022-06-24 12:27:24 id_14 dim: 13
2022-06-24 12:27:24 id_18 dim: 10
2022-06-24 12:27:24 id_19 dim: 100
2022-06-24 12:27:24 card6 dim: 2
2022-06-24 12:27:24 id_21 dim: 100
2022-06-24 12:27:24 id_30 dim: 38
2022-06-24 12:27:24 id_34 dim: 3
2022-06-24 12:27:24 card3 dim: 52
2022-06-24 12:27:24 id_20 dim: 100
2022-06-24 12:27:24 id_09 dim: 24
2022-06-24 12:27:24 id_11 dim: 100
2022-06-24 12:27:24 addr1 dim: 100
2022-06-24 12:27:24 P_emaildomain dim: 30
2022-06-24 12:27:24 Transforming categorical features
